internal/exec/stages/mount/mount.go

```diff
@@ -64,8 +64,14 @@ func (stage) Name() string {
 }
 
 func (s stage) Run(config types.Config) error {
-	fss := config.Storage.Filesystems
-	sort.Slice(fss, func(i, j int) bool { return util.Depth(fss[i].Path) < util.Depth(fss[j].Path) })
+	fss := []types.Filesystem{}
+	for _, fs := range config.Storage.Filesystems {
+		if fs.Path == nil || *fs.Path == "" {
```

Comment:
@ajeddeloh  just to double check this if-statement: if fsPath is nil -don't do anything, if it's not nil - append?  
if that's the case,  checking if it's not nil and appending (and dropping the continue) would be clearer?

Reply:
Just pushed a fix